install.sh

```diff
@@ -187,7 +187,7 @@ done
 sudo ldconfig &> /dev/null || true
 
 # create symlink to 'crew' in ${CREW_PREFIX}/bin/
-ln -sfv "lib/crew/crew" "${CREW_PREFIX}/bin/"
+ln -sfv "../lib/crew/crew" "${CREW_PREFIX}/bin/"
```

Comment:
No, this will break the binary.  There must be something wrong with your environment.

Reply:
Okay, let's think about this step by step.
ln -s is creating a symlink at `#{CREW_PREFIX}/bin/`. bin is a directory, so it uses the name of the file specified in argument 1, crew. So it creates a binary at `#{CREW_PREFIX}/bin/crew`
Then ln looks to the first argument to find out where this symlink should point. It says `lib/crew/crew` so ln makes that the symlinks relative path. Changed into an absolute path, this would be `#{CREW_PREFIX}/bin/lib/crew/crew`. There's no file there, and there isn't even a directory called `lib` in `/usr/local/bin`.
When the shell tries to run `crew`, the symlink points the shell to `/usr/local/bin/lib/crew/crew`, which doesn't exist, so the shell reports that it can't find an executable called `crew`.
If the symlink is instead made to point to `../lib/crew/crew` the symlink then directs the shell to `/usr/local/bin/../lib/crew/crew`, where the file is. If you remove the `bin/..`, that becomes `/usr/local/lib/crew/crew`, which is the correct path.

Reply:
If you really don't believe me do a fresh install with `curl -#L git.io/vddgY | bash` and watch it fail. Then delete everything in `/usr/local` and run `curl -#L git.io/vddgY | sed -e 's:"lib/crew/crew" "${CREW_PREFIX}/bin/":"../lib/crew/crew" "${CREW_PREFIX}/bin/":' | bash` and watch it work.